nimble/host/services/auracast/syscfg.yml

```diff
@@ -0,0 +1,27 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+syscfg.defs:
+  BLE_SVC_AURACAST_ADV_INSTANCE:
+    description: >
+      Defines which advertising instance is used for Auracast.
+    value: 42
```

Comment:
I don't think this should be a syscfg value. app has to call `ble_svc_auracast_create` anyway so it can also include adv instance.

Reply:
as discussed offline, I'd change it to start index of first auracast (BIG). When user creates more, new BIG will occupy <this + offset> instance.

add restriction that additional instances are > 1

Reply:
done